.ci/build.sh

```diff
@@ -1,11 +1,53 @@
 #!/bin/bash
 set -e
 
+# set to one of the following values
+# - don't use HelmQA: (empty)
+# - use HelmQA in SaaS mode: 'http://helmqa-zhaw-prod-demos.appuioapp.ch/livecheck?repo'
+# - use HelmQA locally in Docker mode: 'http://127.0.0.1:5000/livecheck?repo'
+HELMQA_URL='http://helmqa-zhaw-prod-demos.appuioapp.ch/livecheck?repo'
 REPO_URL="${REPO_URL:-https://charts.appuio.ch}"
 echo "----> Deploying to $REPO_URL"
 
 GIT_REPO="$(git config remote.origin.url)"
 
+if [ -n "$HELMQA_URL" ]; then
+  did=
+  if [[ "$HELMQA_URL" =~ 127.0.0.1 ]]; then
+    docker pull jszhaw/helmqa
+    did="$(docker run -d -p 127.0.0.1:5000:5000 helmqa)"
+  fi
+
+  while true; do
+    sleep 1
+    tresponse="$(curl --connect-timeout 5 $HELMQA_URL=$GIT_REPO)" || true
```

Comment:
I would `--max-time` instead of `--connect-timeout`. max-time sets a timeout for ALL parts of the request (connect, sending request, awaiting response, receiving response) while connect-timeout only sets a timeout for the connect phase.

Reply:
These two flags cover different risks. We need connect-timeout with a relatively low value to protect against e.g. firewall DROP rules. On the contrary, max-time needs to consider potentially longer checks. We do not have much data on how much time checks take. I have therefore added max-time additionally with a reasonable default (60s), just to prevent jobs hanging for longer than max HTTP/TCP timeouts. We need some observation to tune all values.